exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -1216,6 +1216,11 @@ public static String bootDefaultFor(String name) {
           "the sender to send out its data more rapidly, but you should know that it has a risk to OOM when the system is solving parallel " +
           "large queries until we have a more accurate resource manager."));
 
+
+  public static final String ENABLE_REST_VERBOSE_ERRORS_KEY = "rest.errors.verbose";
```

Comment:
The name here should be consistent with other similar keys. For example:

```
  public static final String HTTP_ENABLE = "drill.exec.http.enabled";
```

So, maybe, `drill.exec.http.rest.errors.verbose`. Kind of ugly.

We had talked about reworking the keys now that we know what they all are. But, until that is done as a unified task, best to maintain the existing naming conventions so we don't make things more of a mess than they already are.